### PR TITLE
Feature/completing read tag position

### PR DESCRIPTION
**Pull request by marlinstrub**
Hi,

Thanks for your work on this great package.

This PR adds the `org-roam-file-completion-tag-position` option, which can
be used to control the position of `roam_tags` during file completion. It can
take one of the following values

  - `prepend`: Prepends tags to the file title
  - `append`: Appends tags to the file title
  - `omit`: Omits tags during completion.

I prefer to have the tags appended, but `prepend` is left as the default option.

### CODE REVIEW

File: org-roam.el [hunk 1]
```diff
@@ -852,6 +859,24 @@ file."
                  'face 'org-roam-tag))
    str))
 
+(defun org-roam--append-tag-string (str tags)
+  "Append TAGS to STR."
+  (concat
+   str
+   (when tags
+     (propertize (format " (%s)" (s-join org-roam-tag-separator tags))
+                 'face 'org-roam-tag))))
```
Comment (jethrokuan) [inline org-roam.el:868]: Let's merge all these functions into `org-roam--add-tag-string`.

**Review by jethrokuan — COMMENTED** [review]
Can you also add an entry into the changelog?

File: org-roam.el (hunk 1 above)
Comment (marlinstrub) [reply org-roam.el]: Done.

**Comment by marlinstrub** [comment]
Sure. I've updated the changelog.

**Pull request merged**